src/h5web/dataset-visualizer/VisSelector.tsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
-import { FiCpu, FiGrid } from 'react-icons/fi';
+import { FiCpu, FiGrid, FiDisc } from 'react-icons/fi';
 import { IconType } from 'react-icons';
 import { Vis } from './models';
 import styles from './VisSelector.module.css';
 
 const VIS_ICONS: Record<Vis, IconType> = {
   [Vis.Raw]: FiCpu,
+  [Vis.Scalar]: FiDisc,
```

Comment:
I would put `FiCircle` instead of `FiDisc` here

Reply:
Changed to `FiCode`